tests/tests/fv3_ccpp_control

```diff
@@ -86,6 +86,9 @@ export LIST_FILES="atmos_4xdaily.tile1.nc \
 
 export_fv3
 export FHMAX=48
+export NFHOUT=12
```

Comment:
Does this mean model will output fields every 12 hours and there is no high frequency output? If CI test requires, we can change to the previous setting of running fv3_control for 24 hours with high frequency output to 12 hours and then every 3 hours to fh=24h. I will update the global restart tests accordingly. Maybe we can do this in the next commit after Dom's #386.

Reply:
That works for me, Jun.